File: kaushik/kafka_py/producer.py
import random
from kafka import KafkaProducer

producer = KafkaProducer(
    bootstrap_servers=['localhost:9093', 'localhost:9094', 'localhost:9095'])

print("Connected!")

i = random.randrange(10)
if i > 5:
    producer.send(
        topic='cars',
        value=b'Civic',
        partition=1
    )
else:
    producer.send(
        topic='cars',
        value=b'Accord',
        partition=1
    )
print("Message sent!")

producer.close()

File: Iliya/kafka js/producer.js
const {Kafka} = require("kafkajs")



run();



async function run(){
    try
    {
        const kafka = new Kafka({
            "clientId": "yeet",
            "brokers" :["localhost:9093", "localhost:9094", "localhost:9095"]
       })

       const producer = kafka.producer();
       console.log("Connecting.....")
       await producer.connect()
       console.log("Connected!")

       
       const result = await producer.send({

            "topic": "yeet", "messages": [{"value": "2nd msg","partition": 0}]
        });

        console.log(`Message sent: ${JSON.stringify(result)}`);



        console.log("Nothing wrong so far")
        await producer.disconnect();




    }
    catch(ex)
    {
        console.error(`Error code ${ex}`)
    }

    finally{
        process.exit(0);
    }
}